app/controllers/course/condition/achievements_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Course::Condition::AchievementsController < Course::ConditionsController
+  load_resource instance_name: :achievement_condition, class: Course::Condition::Achievement.name,
```

Comment:
It will create `@achievement_condition` using `Course::Condition::Achievement.new` method